packages/path_provider/path_provider_macos/example/pubspec.yaml

```
@@ -1,7 +1,8 @@
 name: path_provider_example
 description: Demonstrates how to use the path_provider plugin.
+publish_to: none
 
-dependencies:
+dependencie:
```

(Please revert this typo)

```suggestion
dependencies:
```

---

Oh, that's why CI is failing. Thanks :)